### PR TITLE
Fail the CI measure process if jest fails

### PR DESCRIPTION
**Pull request by zacharyfmarion**
### Summary

Use the exit code from the jest spawned process to fail the reassure job if jest errors out. Our performance tests were silently failing since jest was failing to properly render the components. See https://github.com/callstack/reassure/issues/171

### Test plan

I tested locally by manually throwing an error in a test and running reassure.

### CODE REVIEW

File: packages/reassure-cli/src/commands/measure.ts [hunk 1]
```diff
@@ -35,7 +35,11 @@ export function run(options: MeasureOptions) {
     { shell: true, stdio: 'inherit', env: { ...process.env, OUTPUT_FILE: outputFile } }
   );
 
-  console.log('');
```
Comment (zacharyfmarion) [inline packages/reassure-cli/src/commands/measure.ts:38]: Not sure if this was intentional?

Comment (mdjastrzebski) [reply packages/reassure-cli/src/commands/measure.ts:38]: This `console.log` was most probably inserting a new line there. Could you please restore it:
```ts
console.log(''); // insert newline
```

File: packages/reassure-cli/src/commands/measure.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export function run(options: MeasureOptions) {
   const outputFile = options.baseline ? BASELINE_FILE : RESULTS_FILE;
   rmSync(outputFile, { force: true });
```
Comment (mdjastrzebski) [inline packages/reassure-cli/src/commands/measure.ts:24]: const testRunnerPath = process.env.TEST_RUNNER_PATH ?? 'node_modules/.bin/jest';

File: packages/reassure-cli/src/commands/measure.ts [hunk 3]
```diff
@@ -35,7 +35,11 @@ export function run(options: MeasureOptions) {
     { shell: true, stdio: 'inherit', env: { ...process.env, OUTPUT_FILE: outputFile } }
   );
 
-  console.log('');
+  if (spawnInfo.status !== 0) {
+    console.error(`❌  Something went wrong, jest process exited with an error`);
```
Comment (mdjastrzebski) [inline packages/reassure-cli/src/commands/measure.ts:39]:
```suggestion
    console.error(`❌  Test runner (${testRunnerPath}) exited with error code ${spawnInfo.status}`);
```

**Review by mdjastrzebski — CHANGES_REQUESTED** [review]
Looks good! Suggested some small tweaks.

**Comment by zacharyfmarion** [comment]
@mdjastrzebski thanks for the quick review - addressed feedback!

**Review by thymikee — APPROVED** [review]

**Pull request merged**